MBBSEmu.Tests/ExportedModules/Majorbbs/stzcpy_Tests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class stzcpy_Tests : ExportedModuleTestBase
+    {
+        private const int STZCPY_ORDINAL = 589;
+
+        // DST buffer is "AAAAAAAA"
+        [Theory]
+        [InlineData(8, "Test", 4, "TestAAAA")]
+        [InlineData(8, "Test", 0, "AAAAAAAA")]
+        [InlineData(8, "Test", 1, "TAAAAAAA")]
+        [InlineData(8, "Test", 5, "Test")]
+        [InlineData(12, "TestTestTest", 12, "TestTestTest")]
+        public void stzcpy_Test(ushort dstLength, string srcString, ushort copyLength, string expected)
+        {
+            //Reset State
+            Reset();
+
+            //Set Argument Values to be Passed In
+            var destinationStringPointer = mbbsEmuMemoryCore.AllocateVariable("DST", dstLength);
+            mbbsEmuMemoryCore.SetArray("DST", Enumerable.Repeat((byte) 'A', dstLength).ToArray());
+
+            var sourceStringPointer = mbbsEmuMemoryCore.AllocateVariable("SRC", (ushort) (srcString.Length + 1));
+            mbbsEmuMemoryCore.SetArray("SRC", Encoding.ASCII.GetBytes(srcString));
+
+            //Execute Test
+            ExecuteApiTest(HostProcess.ExportedModules.Majorbbs.Segment, STZCPY_ORDINAL,
+                new List<ushort>
+                {
+                    destinationStringPointer.Offset,
+                    destinationStringPointer.Segment,
+                    sourceStringPointer.Offset,
+                    sourceStringPointer.Segment,
+                    copyLength
```

Comment:
dstLength is the arg to stzcpy, e.g. stzcpy(char *dst, char *src, int dstLength);

We also want to verify that any remaining bytes in dst are zero filled at the end, e.g. if you do this

```
char buf[32];
stzcpy(buf, "test", sizeof(buf));
```

buf would have the string "test" and all remaining zeros after it. So validating the zeros is an important assertion to make for this API call